src/coreclr/gc/gc.cpp

```diff
@@ -43154,6 +43173,71 @@ unsigned int GCHeap::WhichGeneration (Object* object)
     return g;
 }
 
+unsigned int GCHeap::WhichRange (Object* object, uint8_t** ppStart, uint8_t** ppAllocated, uint8_t** ppReserved)
```

Comment:
>WhichRange [](http://example.com/codeflow?start=21&length=10)

I would probably call this GetGenerationWithRange because generation is an important piece of info you are returning.